lib/features/mailbox/presentation/widgets/mailbox_item_widget.dart

```diff
@@ -77,59 +77,70 @@ class _MailboxItemWidgetState extends State<MailboxItemWidget> {
+              horizontal: MailboxItemWidgetStyles.itemPadding,
+            ),
+            height: widget.itemHeight ??
+                (widget.mailboxNode.item.isTeamMailboxes
                     ? MailboxItemWidgetStyles.teamMailboxHeight
                     : MailboxItemWidgetStyles.height),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  crossAxisAlignment: widget.mailboxNode.item.isTeamMailboxes
-                    ? CrossAxisAlignment.start
-                    : CrossAxisAlignment.center,
-                  children: [
-                    if (_isIconDisplayed)
-                      MailboxIconWidget(
-                        icon: _iconMailbox,
-                        color: widget.iconColor,
-                      ),
-                    Expanded(
-                      child: LabelMailboxItemWidget(
-                        itemKey: _key,
-                        mailboxNode: widget.mailboxNode,
-                        isItemHovered: _isItemHovered,
-                        isSelected: _isSelected,
-                        isHighlighted: widget.isHighlighted,
-                        textStyle: widget.textStyle,
-                        onMenuActionClick: widget.onMenuActionClick,
-                        onEmptyMailboxActionCallback: widget.onEmptyMailboxActionCallback,
-                        onClickExpandMailboxNodeAction: widget.onExpandFolderActionClick,
-                      )
-                    ),
-                  ]
-                )
-              ),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: widget.mailboxNode.item.isTeamMailboxes
+                  ? CrossAxisAlignment.start
+                  : CrossAxisAlignment.center,
+              children: [
+                if (_isIconDisplayed)
+                  MailboxIconWidget(
+                    icon: _iconMailbox,
+                    color: widget.iconColor,
+                  ),
+                Expanded(
+                  child: LabelMailboxItemWidget(
+                    itemKey: _key,
+                    mailboxNode: widget.mailboxNode,
+                    isItemHovered: _isItemHovered,
+                    isSelected: _isSelected,
+                    isHighlighted: widget.isHighlighted,
+                    textStyle: widget.textStyle,
+                    onMenuActionClick: widget.onMenuActionClick,
+                    onEmptyMailboxActionCallback:
+                        widget.onEmptyMailboxActionCallback,
+                    onClickExpandMailboxNodeAction:
+                        widget.onExpandFolderActionClick,
+                  ),
+                ),
+              ],
             ),
-          );
+          ),
+        ),
+      );
+
+      if (widget.mailboxNode.item.isActionRequired) {
+        return itemWidget;
+      }
+
+      return DragTarget<List<PresentationEmail>>(
```

Comment:
should not show inkwell effect when user drag email into virtual folder

Reply:
Fixed